dist/redhat/scylla-cqlsh.spec

```diff
@@ -3,7 +3,7 @@ Version:        %{version}
 Release:        %{release}%{?dist}
 Summary:        cqlsh is a Python-based command-line client for running CQL commands on a cassandra cluster.
 Group:          Applications/Databases
-Provides:       %{product}-cqlsh
+Provides:       %{product}-cqlsh = %{version}-%{release}
```

Comment:
Do we even need this line? I thought a package automatically provides name-version-release.

There's the ${?dist}, but we could just add it to the Requires line, or just drop it.

Since there are multiple options, I'm okay with applying this and thinking about it later. But waiting for @syuu1228 ack.

Reply:
indeed, verified:
```console
$ rpm -qp --provides /home/kefu/dev/scylladb/tools/python3/build/redhat/RPMS/x86_64/scylla-python3-5.3.0~rc1-7a3c64d41c99.x86_64.rpm
scylla-python3 = 5.3.0~rc1-7a3c64d41c99   
scylla-python3(x86-64) = 5.3.0~rc1-7a3c64d41c99
```